EventFilter/EcalRawToDigi/plugins/EcalRawToDigiGPU.cc

```diff
@@ -138,6 +138,10 @@ void EcalRawToDigiGPU::acquire(edm::Event const& event,
   if (counter > 0) {
     ecal::raw::entryPoint(
         inputCPU, inputGPU, outputGPU_, scratchGPU, outputCPU_, conditions, ctx.stream(), counter, currentCummOffset);
+  } else {
```

Comment:
I think it would be cleaner to set these to 0 unconditionally, before calling (or not) `ecal::raw::entryPoint`.

Reply:
Can be done as well but zeroing seems an unnecessary operation when `ecal::raw::entryPoint` is called.

Reply:
The idea is to make sure the variables are in a clean (initialised) state, independently of the future use.
Especially because they are only set in the subsequent call to `produce()`, and anybody looking at them inside `ecal::raw::entryPoint` (for example, to debug the next problem in a few weeks/months/years time) will likely be confused by seeing random values.